web/components/templates/dashboard/dashboardPage.tsx

```diff
@@ -959,50 +1028,133 @@ const DashboardPage = (props: DashboardPageProps) => {
                 </StyledAreaChart>
               </div>
 
-              <div key="threats">
+              <div key="quantiles">
+                <Card>
+                  <div className="flex flex-row items-center justify-between">
+                    <div className="flex flex-col space-y-0.5">
+                      <p className="text-gray-500 text-sm">Quantiles</p>
+                      <p className="text-black dark:text-white text-xl font-semibold">
+                        {`Max: ${new Intl.NumberFormat("us").format(
+                          max(
+                            quantiles?.data
+                              ?.map((d) => d.p99)
+                              .filter((d) => d !== 0) ?? []
+                          )
+                        )} ${currentMetric === "Latency" ? "ms" : ""}`}
+                      </p>
+                    </div>
+                    {!quantilesIsLoading && (
```

Comment:
nit: if there is ever a component that is has state (currentMetric) that only applies to itself, make it into its own self-contained component so that this state change does not refresh the entire component